react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import './ValidatedInput.css'
+
+export default function ValidatedInput() {
+  const [passwordEntry, setPasswordEntry] = useState('');
+
+  function passwordChange(e) {
+    setPasswordEntry(e.target.value)
+  }
+  let verified = false
```

Comment:
The code might be more clear if you used the name `error` instead of `text`. Then it would make sense to just check `error` instead of using this additional `verified` variable.